app/src/main/java/de/westnordost/streetcomplete/quests/drinking_water/AddDrinkingWater.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.drinking_water
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+
+class AddDrinkingWater : OsmFilterQuestType<DrinkingWater>() {
+
+    override val elementFilter = """
+        nodes with (
+            man_made = water_tap
+            or man_made = water_well
+            or amenity = toilets
```

Comment:
should remove that. Too common and it will be always the same answer that depends on the region. ("Yes, no sign" or "no, no sign")